### PR TITLE
 Add username and password options 

### PR DESCRIPTION
**Pull request by zschwarz**

### CODE REVIEW

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java [hunk 1]
```diff
@@ -173,9 +183,13 @@ void printMessage(String topic, MqttMessage message) {
         }
     }
 
-    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions) {
-        connectOptions.setUserName("admin");
-        connectOptions.setPassword("admin".toCharArray());
+    static MqttConnectOptions setConnectionOptions(MqttConnectOptions connectOptions, String username, String password) {
```
Comment (jdanekrh) [inline cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java]: Depending on how many more parameters we may need to add in the future, we should maybe do this differently. It is not good if a method takes more than 5 or so parameters.

**Comment by kornys** [comment]
@jdanekrh I don't think that having a too many commits is bad, because you, like we in enmasse, use  "squash and merge".

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java (hunk 1 above)
Comment (michalxo) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Client.java]: Let's see what would happen in the future. Let's approve this one for now.

**Comment by michalxo** [comment]
@zschwarz please have a look at Codacy/PR quality review. This one suggested to not use package-private by default, but private (unless needed otherwise). [optional]

CodeClimate noticed in Sender & Receiver client that you use the very same code. Seems like you should use 1 method for them? Maybe "Utils.java" Checkers.java file/class?

**Comment by michalxo** [comment]
Also, this PR is about adding a username and password, but there is a commit for "will" messages as well. It is a bit misleading, so in future, please create 2 separate PRs or name this one differently please :-)

**Comment by jdanekrh** [comment]
>The PR contains too many commits.
>>@jdanekrh I don't think that having a too many commits is bad, because you, like we in enmasse, use "squash and merge".

The problem was that the pr also carried different versions of commits that were already on master from a different pr.

**Pull request closed without merging**